### PR TITLE
add try/catch

### PR DESCRIPTION
**Pull request by dothebart**
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8604/

### CODE REVIEW

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8606/

File: tests/js/common/shell/shell-collection-rocksdb-noncluster.js [hunk 1]
```diff
@@ -408,102 +408,106 @@ function CollectionSuite () {
...
+        idxs = c.getIndexes(true);
+        // cache was partially filled same queries, lifetime hit rate
+        // should be about 50 %
+        idxs.forEach(function(idx) {
+          if (idx.figures.cacheInUse) {
+            assertTrue(idx.figures.cacheLifeTimeHitRate > 15, idx);
+          }
+        });
+      } catch (x) {
+        db._drop(x);
```
Comment (jsteemann) [inline tests/js/common/shell/shell-collection-rocksdb-noncluster.js]: should be `db._drop(cn);`, not `db._drop(x);`.
However, you use try..catch here and not try..finally?
That would be a lot better!

**Comment by dothebart** [comment]
tests all green.

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8611/

**Pull request merged**